Makefile

```
@@ -0,0 +1,22 @@
+.PHONY: all build
+
+all: build
+
+
+#  __
+# |__)     . |  _|
+# |__) |_| | | (_|
+#
+
+build: clean
+	GOOS=darwin GOARCH=amd64 go build -o ./build/macos-amd64/splitter-service
+
+clean:
+	rm -rf build
+
+test:
+	go test ./...
+
+deps:
+	go mod download
+	go mod verify
```

Can we add the following commands as well to the file please?
I think it would be useful:

```suggestion
	go mod verify
	
.PHONY: lint
lint:
	golangci-lint run ./... -v

.PHONY: fmt
fmt:
	go fmt ./...

.PHONY: run
run:
	cd build
	go run splitter-service
```